12-Capstone/aws-eb.md

```diff
@@ -0,0 +1,118 @@
+# Amazon Web Services & Elastic Beanstalk
+
+In this lesson you will learn about Amazon Web Services in general and the Elastic Beanstalk Service in particular.
+
```

Comment:
What is the intent of this resource? As a reading (we already have a lot in the homework for tonight) or as a lecture?